gym/wrappers/record_episode_statistics.py

```diff
@@ -7,11 +7,77 @@
 import gym
 
 
+class EnvStatsInfo:
```

Comment:
I'm not sure I like the way these two classes are structured in general. The only functionality which is not a one-liner is `VectorEnvStatsInfo.add_episode_statistics`. Wouldn't it be possible to just maintain the `self.info` dictionary inside of `RecordEpisodeStatistics`, and then in the code it'd be updated with the one-liners I mentioned, and `add_episode_statistics` which would be a separate function?

Reply:
So I've simplified things a lot, got rid of the classes, and just left the function to process episode_statistics. Previously the idea was to have the same interface for both vectorized and non vectorized envs and then choose the processing strategy depending on the type of env (this was the reason of some unused args in the non vectorized envs class)